### ISSUE TITLE
Proposal: Using accessibility service to build one of a kind User Experience

### ISSUE THREAD

**Issue by rohanagarwal94** [gdgnewdelhi/talks#13]
Please fill out the form below with respect to your proposed talk before submitting, thank you!

- Using accessibility service to deliver one heck of a user experience.
- Accessibility service apart from helping people with disabilities can also help companies deliver truly enriching user experience. Examples including voodoo and freecharge mobile recharge on whatsapp.
- Key takeaways - Accessiblity service, Custom Views, Overlays, Accessibility Events, Drawing over Apps
- 1-2 hr
- Intermediate
- Laptops with min 8 GB RAM, Android Studio, and Android SDK with all things configured.

**Comment by nisrulz** [comment]
@rohanagarwal94 we are going to have a meetup soon, would you be available?

**Comment by rohanagarwal94** [comment]
@nisrulz totally depends on the dates. My exams are over on 19th. I have prior commitments in between. Probably any day after 23rd May.

**Comment by nisrulz** [comment]
We plan to do the meetup on either of the dates: 27 or 28 May. Does that work for you?

**Comment by rohanagarwal94** [comment]
I am 99% certain that it will work out for me. However just to be on the safer side, is it fine for you if I confirm on the 19th itself?

**Comment by nisrulz** [comment]
Yup. Works for us. All the best for your exams 😄

**Comment by rohanagarwal94** [comment]
Thanks man 😇

**Comment by rohanagarwal94** [comment]
@nisrulz wanted to confirm about the talk. Is 28th doable?. I am occupied on the 27th.

**Comment by nisrulz** [comment]
@rohanagarwal94 we are having some issues regarding the venue for now. We will update you as and when we have some information. We might be moving the meetup into June I think.

**Comment by nisrulz** [comment]
@rohanagarwal94 We think we have fixed it all up, are you still available for 28th? IF yes, we have a pre-condition. We can only give you 1 hr slot. Do you thing you can adjust your talk for that slot?

Please update us asap, as the date is near.

**Comment by rohanagarwal94** [comment]
@nisrulz yeah that only depends on the venue and time slot. Could you be more specific on that front?

**Comment by arpansac** [comment]
Hi @rohanagarwal94 
These are the details: https://www.meetup.com/GDGNewDelhi/events/240141035/
10 am to 12pm
Please replay ASAP :)

**Comment by rohanagarwal94** [comment]
@arpansac sounds good to me.

**Comment by arpansac** [comment]
We're still finalizing 1 more speaker, I'll get back to you by the end of the day if confirmed!

**Comment by arpansac** [comment]
Hi @rohanagarwal94 , unfortunately we couldn't get the date of the other speaker, we've postponed the event to the next weekend (tentatively) But I'm kind of certain that we'll have it on that date. Would you be available on next Sunday (4th June, 2017)?

**Comment by rohanagarwal94** [comment]
Yeah I am available next Sunday.

**Comment by arpansac** [comment]
@rohanagarwal94 Should I confirm it for Sunday then?

**Comment by arpansac** [comment]
If yes, then please ping the organizers on our meetup.com page with your email id and I'll take it forward from there!

**Comment by rohanagarwal94** [comment]
Yes yes. Sorry am I late?
